mopidy_soundcloud/library.py

```diff
@@ -38,14 +40,19 @@ class SoundCloudLibraryProvider(backend.LibraryProvider):
         uri="soundcloud:directory", name="SoundCloud"
     )
 
-    def __init__(self, *args, **kwargs):
+    def __init__(self, web_client, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.vfs = {"soundcloud:directory": {}}
         self.add_to_vfs(new_folder("Following", ["following"]))
         self.add_to_vfs(new_folder("Liked", ["liked"]))
         self.add_to_vfs(new_folder("Sets", ["sets"]))
         self.add_to_vfs(new_folder("Stream", ["stream"]))
 
+        self.image_provider = SoundCloudImageProvider(web_client)
+
+    def get_images(self, uris):
+        return self.image_provider.get_images(uris)
```

Comment:
I can't see where this method is called.

Reply:
This method is called from this line here: https://github.com/mopidy/mopidy/blob/ffe962742aeb4837f475a68bf82e61780591ca76/mopidy/core/library.py#L171

Reply:
Ah, that makes sense :+1: